src/lib/full-path.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Get a PostCSS source full path
+ * @module lib/full-path
+ */
+
+const {join, dirname} = require('path');
+
+module.exports = (fp, rp) =>
```

Comment:
Depends. I've split the code in 4 rows because I thought that it could be more readable for other people. A one-liner it's quite clear to me.

Just one thing, please: leave the new line after the fat-arrow; it makes the function signature clearer